teams/teams-ps/teams/New-CsOnlineLisLocation.md

```diff
@@ -395,7 +372,7 @@ Accept wildcard characters: False
 ### -StreetSuffix
```

Comment:
Looks like description about -Elin is missing in this cmdlet. can we add description

-Elin
Specifies the Emergency Location Identification Number. This is used in Direct Routing EGW scenarios.

Reply:
param was missing in cmdlet. I added that. But description is there.